cmd/root.go

```diff
@@ -34,6 +34,11 @@ var rootCmd = &cobra.Command{
 			Image:          containerImage,
 			EnabledChecks:  parseEnabledChecksValue(),
 			ResponseFormat: parseOutputFormat(),
+			CLILogFile:     parseCLILogFile(),
+		}
+
+		if err := initializeLoggerForConfig(cfg); err != nil {
```

Comment:
nit - could we name this `initLogger` or even `initLog'?  Under the premise that the ForConfig suffix doesn't make it more readable.

Reply:
@tonytcampbell Fair - I think my goal here with the naming was to make it clear that we were configuring our logger based on the user configuration. In other places, the `NewForConfig(runtime.Config)` semantic is used (e.g. [engine](https://github.com/redhat-openshift-ecosystem/openshift-preflight/blob/6a626e2306c2eb6d25600d57611dc333b425f2cd/certification/engine/engine.go#L37)).

I couldn't do that here because the logger configuration isn't in its own package, and putting in its own package when there is a minimal amount of action taking place seemed silly at the time.

Summarizing: I can change it, but it would differ a bit from the semantic. It doesn't match it exactly anyway! If you think moving it to its own package would make it more readable while maintaining the "<Action>ForConfig" semantic (something like `logger.ProvisionForConfig(runtime.Config)` or similar), I could do that as well.

Let me know!

Reply:
I've changed this to `initLogger` in the interim. Happy to move forward with that or change it to whatever works best!